Source/WebCore/css/calc/CSSCalcOperationNode.cpp

```diff
@@ -1369,6 +1370,14 @@ double CSSCalcOperationNode::evaluateOperator(CalcOperator op, const Vector<doub
     case CalcOperator::Tan: {
         if (children.size() != 1)
             return std::numeric_limits<double>::quiet_NaN();
+        double x = std::fmod(children[0], piDouble * 2);
+        // std::fmod can return negative values.
+        x = x < 0 ? piDouble * 2 + x : x;
```

Comment:
I’m a bit surprised we don’t call `std::tan(x)` below.

Reply:
That works too.